resources/views/admin/bans/new.blade.php

```diff
@@ -0,0 +1,90 @@
+@extends('layouts.app')
+@section('content')
+    @component('components.errors')
+    @endcomponent
+
+    <div class="card">
+        <h5 class="card-header">Add ban</h5>
+        <div class="card-body">
+            {{ Form::open(['url' => route('admin.bans.create'), 'method' => 'POST']) }}
+            {{ Form::token() }}
+
+            <div class="form-group mb-4">
+                Ban type
+                <div class="custom-control custom-radio">
+                    {{ Form::radio('ip', 0, old('ip') === 0, ['class' => 'custom-control-input', 'id' => 'ip-0']) }} {{ Form::label('ip-0', 'Username or IPv6 Address', ['class' => 'custom-control-label']) }}
```

Comment:
This isn't accurate now as it will also match on an on-wiki IPv4 block, how about `Username or IP address being appealed`

Reply:
The problem is if they select this option, the IP address is not considered that is pulled from the server. It allows ban evasion by not checking the server IP. Aka I'm 192.168.1.1 but I appeal for 172.0.0.1 knowing full well my 192.168.1.1 range is already banned. We've had people pull this off in one or both UTRS versions.